packages/web/app/src/components/v2/diff-editor.tsx

```diff
@@ -49,9 +52,9 @@ export const DiffEditor = ({
   }
 
   return (
-    <div className="h-full w-full">
+    <div className={cx('h-full w-full', className)}>
       <div className="flex items-center justify-between p-5">
-        <div className="font-semibold">{title}</div>
+        {title && <div className="font-semibold">{title}</div>}
```

Comment:
I can revert it now that I don't need `DiffEditor` after all, but I think it might be a nice-to-have API change for this component.